src/content/guides/prepare-aws-account-for-tenant-clusters/index.md

```diff
@@ -5,6 +5,13 @@ date: 2020-05-19
 type: page
 weight: 100
 tags: ["tutorial"]
+user_questions:
+  - Where can I find the AWS account ID to use for running a installation?
+  - Where I can find the AWS account ID to run tenant clusters?
+  - What are the recommended service limit/quotas for AWS accounts used with Giant Swarm?
+  - How to see this in AWS console itself for the existing installation?
+  - How to gather this data in China?
+  - How can I enable the new network services account on the Control Plane?
```

Comment:
I took this from the spreadsheet and made each question one line. The questions still need some work IMO. "this data" is not understandable this way. "the new network services account" does not speak to me.

Reply:
Decided to remove those.